tests/lib/rules/comma-spacing.js

```diff
@@ -198,17 +240,6 @@ ruleTester.run("comma-spacing", rule, {
                 }
             ]
         },
-        {
-            code: "var arr = [1 , ];",
-            output: "var arr = [1 ,];",
-            options: [{ before: true, after: false }],
-            errors: [
-                {
-                    message: "There should be no space after ','.",
-                    type: "Punctuator"
-                }
-            ]
-        },
```

Comment:
This was conflicting with `array-bracket-spacing` option `"always"` on the right side of `,`.

[Playground link](https://eslint.org/play#eyJ0ZXh0IjoiLyogZXNsaW50IGNvbW1hLXNwYWNpbmc6IFtcImVycm9yXCIsIHsgYmVmb3JlOiB0cnVlLCBhZnRlcjogZmFsc2UgfV0gKi9cbi8qIGVzbGludCBhcnJheS1icmFja2V0LXNwYWNpbmc6IFtcImVycm9yXCIsIFwiYWx3YXlzXCJdICovXG5cbnZhciBhcnIgPSBbMSAsIF07XG5cbiIsIm9wdGlvbnMiOnsicGFyc2VyT3B0aW9ucyI6eyJlY21hVmVyc2lvbiI6ImxhdGVzdCIsInNvdXJjZVR5cGUiOiJtb2R1bGUiLCJlY21hRmVhdHVyZXMiOnt9fSwicnVsZXMiOnt9LCJlbnYiOnt9fX0=)